hieradata_aws/class/jenkins.yaml

```diff
@@ -53,6 +53,7 @@ govuk_jenkins::config::user_permissions:
 govuk_jenkins::config::banner_string: 'AWS Deploy'
 
 govuk_jenkins::job_builder::jobs:
+  - govuk_jenkins::job::data_sync_complete
```

Comment:
I think this will need to be `govuk_jenkins::job::data_sync_complete_integration`

Reply:
Ah!